src/app/features/public-form/public-form.module.ts

```diff
@@ -76,7 +83,14 @@ const viewConfigs: ViewConfig[] = [
               config: {
                 fieldGroups: [
                   {
-                    fields: ["route", "title"],
+                    fields: [
+                      {
+                        id: "route",
+                        editComponent: "EditPublicformRouteComponent",
```

Comment:
does this also work if you add the editComponent to the PublicFormConfig entity class instead? Then everything is in one place

Reply:
I already tried but this is not working when adding it to entity class

Reply:
It seems to work for me, if I move it to PublicFormConfig.

Could it be that your local database in the Config:... somehow contains a custom "entity:PublicFormConfig" definition and therefore the code change does not take effect?

Reply:
thanks, Sebastian, I checked my local database and found Config:CONFIG_ENTITY still has old definitions related to the route field so I updated that, and it's working for me too.